src/controllers/dlgprefcontroller.cpp

```diff
@@ -1150,21 +1143,21 @@ void DlgPrefController::showMapping(std::shared_ptr<LegacyControllerMapping> pMa
         m_pMapping = pMapping;
     }
 
-#ifdef MIXXX_USE_QML
     // Add the screens tab if there are screens
     if (pMapping && CmdlineArgs::Instance().getControllerPreviewScreens()) {
+#ifdef MIXXX_USE_QML
```

Comment:
I don't think this makes sense. If a user set `--controller-preview-screens` args, there will be presented with an empty, broken tab.

Reply:
Done!